plugins/builddep.py

```diff
@@ -216,6 +216,11 @@ def _remote_deps(self, package):
                 name=(sourcenames + [package]), arch="src").latest().run()
         if not pkgs:
             raise dnf.exceptions.Error(_('no package matched: %s') % package)
-        self.base.download_packages(pkgs)
+        done = True
         for pkg in pkgs:
-            self._src_deps(pkg.localPkg())
+            for req in getattr(pkg, 'requires'):
```

Comment:
Can you change it to `pkg.requires` then, please. I find it more readable and so better for maintenance.